src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -1850,9 +1850,6 @@
         - name: --roles
           type: string
           short-summary: Specify the space-separated roles.
-        - name: --source-resource-id -s
-          type: string
-          short-summary: Specify the source resource id of the binding.
```

Comment:
May I ask deleting the `--source-resource-id` parameter for the `aks trustedaccess rolebinding update` command will cause breaking change to users?

Reply:
What does this "breaking change" means? From server side we already deny the request of updating source-resource-id, so if users use old version CLI and specify the --source-resource-id, it will get error unless he remove --source-resource-id option.

Reply:
Got it, thanks~

Reply:
For CLI breaking change, please refer to this doc: [breaking_changes.md](https://msazure.visualstudio.com/One/_git/AzureCliTools-Wiki?version=GBmain&path=/src/teams_docs/azcli_docs/breaking_changes.md)